### PR TITLE
Update pools

### PR DESCRIPTION
**Pull request by dogefy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -244,13 +244,13 @@ Please contact us if you have a new mining pool.
 - 2pool@tokyo: https://2pool.sugarchain.org
 - rplant@worldwide: https://pool.rplant.xyz
 - mofumofu@tokyo: https://nomp.mofumofu.me
-- dxpool@china: https://www.dxpool.com
-- anomp@china: http://39.103.144.156
+- dxpool@china: https://www.dxpool.net/?pool=labs
```
Comment (decryp2kanon) [inline README.md:247]: it seems `https://www.dxpool.com/?pool=labs` still works

File: README.md [hunk 2]
```diff
@@ -244,13 +244,13 @@ Please contact us if you have a new mining pool.
...
+- anomp@china: https://anomp.cc/getting_started/sugar
 - customspeed@netherlands: http://pool.customspeed.nl
 - cpu-pool@russia: http://cpu-pool.com
 - zpool@worldwide: https://zpool.ca/algo/yespowerSUGAR
 - zergpool@worldwide: https://zergpool.com/site/mining?algo=yespowerSugar
-- sugarchain.eu@czech republic: https://sugarchain.eu
+- ~~sugarchain.eu@czech republic: https://sugarchain.eu~~
```
Comment (decryp2kanon) [inline README.md:253]: seems not working

**Review by decryp2kanon — CHANGES_REQUESTED** [review]
https://sugarchain.eu seems not working

File: README.md (hunk 1 above)
Comment (dogefy) [reply README.md]: dxpool.com is not available in some areas, so add dxpool.net instead

File: README.md (hunk 2 above)
Comment (dogefy) [reply README.md]: Removed

**Review by decryp2kanon — APPROVED** [review]
thanks

**Comment by decryp2kanon** [comment]
fixed thanks!

image

**Pull request merged**